### ISSUE TITLE
Low Carb (Carbohydrate) Nuts

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#84]
I've decided to carry nuts with me at _all_ times so I can snack on something _nutritious_ without resorting to "_junk_". Did a bit of searching and found this good comparison:

https://www.dietdoctor.com/low-carb/nuts
![image](https://user-images.githubusercontent.com/194400/35919866-2c63c3b2-0c0e-11e8-9a02-3d06a1782c94.png)

+ Pecan nuts – 100 g (3½ ounces or about three handfuls) contains 4 grams of net carbs.
+ Brazil nuts – 100 g contains 4 grams of net carbs.
+ Macadamia – 100 g contains 5 grams of net carbs.
+ Peanuts – 100 g contains 7 grams of net carbs.
+ Hazel nuts – 100 g contains 7 grams of net carbs.
+ Walnuts – 100 g contains 7 grams of net carbs.
+ Almonds – 100 g contains **10 grams** of net carbs.

![image](https://user-images.githubusercontent.com/194400/35919945-55864c10-0c0e-11e8-9cbd-aa5559d769ac.png)

> I'm _sad_ because my Cashews are _by far_ my favourite nuts! 😞 
> It just means I can't eat _only_ cashews ... gotta mix them up!

Going to prepare **50g** portions of _mixed_ nuts with a heavy bias towards the lower carb (_sadly, more expensive!_) nuts. 

Going to figure out how much Organic Nuts cost in _bulk_. 
First port of call: https://www.buywholefoodsonline.co.uk/nuts

**Comment by iteles** [comment]
Amazon is sometimes very good price-wise for bulk nuts (and coconut chips).

**Comment by nelsonic** [comment]
@iteles yeah, I've checked AMZN and prices are "_reasonable_".

Also, I will be including **Coconut Chips** which are also Low Carb:
https://www.perfectketo.com/nuts-on-a-ketogenic-diet/
![coconut-chips-low-carb](https://user-images.githubusercontent.com/194400/35927549-ef0023e8-0c22-11e8-87fd-00f08c31083d.png)

Nutrition:
![image](https://user-images.githubusercontent.com/194400/35927622-0ca31f5e-0c23-11e8-9fbf-b48c856a955b.png)
Super high-calorie (_**663 Kcal** per 100g!!_) but crucially Low Carb!

https://www.buywholefoodsonline.co.uk/organic-coconut-chips-raw-1kg.html
![image](https://user-images.githubusercontent.com/194400/35927385-8082cd1c-0c22-11e8-8647-5915eb7d82bb.png)

The _same_ product on AMZN is more expensive: https://www.amazon.co.uk/Buy-Whole-Foods-Organic-Coconut/dp/B007VOV84U
![image](https://user-images.githubusercontent.com/194400/35927472-b8bc4f28-0c22-11e8-8347-1b6493a998d2.png)

**Comment by iteles** [comment]
(Personal note: we still have about 600g of coconut chips at home so no need to buy these straight away)

**Comment by nelsonic** [comment]
@iteles yeah, we're "all set" for Coconut Chips and Almonds for the next few weeks.

I want to _test_ adding cocoa powder and salt (_without any sweetener_) to _roast_ nuts... 🤔

**Comment by rub1e** [comment]
> I want to test adding cocoa powder and salt (without any sweetener) to roast nuts...

I'd like live updates from this experiment please 

And if you should find a technique which makes the salt stick to the almonds, I'd also be glad to hear about it 😋
